glue/core/data_exporters/gridded_fits.py

```diff
@@ -9,6 +9,22 @@
 __all__ = []
 
 
+def make_component_header(component, header):
+    """
+    Function that extracts information from components
+    and adds it to the data header. The input header is
+    expected to come from Data.coords.header by default.
+    :param component: glue Component
+    :param header: astropy.io.fits.header.Header
+    :return:
```

Comment:
I'm currently trying to switch over to the Numpydoc format (the same one used by astropy), so could you update this docstring accordingly?